rules/windows/privilege_escalation_unusual_svchost_childproc_childless.toml

```diff
@@ -0,0 +1,67 @@
+[metadata]
+creation_date = "2020/10/13"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies unusual child processes of svchost services that normally do not spawn any child processes. This may indicate
```

Comment:
```suggestion
Identifies unusual child processes of Service Host (svchost.exe) that traditionally do not spawn any child processes. This may indicate
```